content/blog/2016-06-21-july-meeting.md

```diff
@@ -7,9 +7,9 @@ Summary: Dave wrangles databases with python; debut tech-collective.com; collabo
 
 The next meeting will take place at:
 
-6pm  
```

Comment:
@anlek does your editor remove trailing whitespace? i think in mardkwon, if there's not 2 spaces at line end, there will be no line break.